modules/nw-ovn-ipsec-north-south-enable.adoc

```diff
@@ -209,6 +209,42 @@ A successfully updated node has the following status: `UPDATED=true`, `UPDATING=
 By default, the MCO updates one machine per pool at a time, causing the total time the migration takes to increase with the size of the cluster.
 ====
 
+. To confirm that IPsec machine configs rolled out successfully, enter the following commands:
++
+[source,terminal]
+----
+$ oc get mc | grep ipsec
+----
++
+.Example output
+[source,text]
+----
+80-ipsec-master-extensions        3.2.0        6d15h
+80-ipsec-worker-extensions        3.2.0        6d15h
+----
++
+[source,terminal]
+----
+$ oc get mcp master -o yaml | grep 80-ipsec-master-extensions -c 2
```

Comment:
The command is just `oc get mcp master -o yaml | grep 80-ipsec-master-extensions -c` and output must be `2`